datafusion/physical-plan/src/sorts/merge.rs

```diff
@@ -97,6 +100,10 @@ pub(crate) struct SortPreservingMergeStream<C: CursorValues> {
 
     /// number of rows produced
     produced: usize,
+
+    /// Unitiated partitions. They are stored in a vector to keep them in
```

Comment:
Can you please document what an "uninitiated partition" means in this context? I think it means  partitions whose streams that have been polled haven't been ready yet